docs/usage.md

```diff
@@ -91,3 +93,143 @@ Find and set the options which fit to your demands, it is also an option to chan
+
+These steps need an amount of user interaction which can be overcome with the automatic base installation option described in this section.  
+The automatized setup is based on the configuration file `/boot/dietpi.txt`. It can be edited prior to the first boot and will be evaluated during the first boot procedure. On subsequent boot procedures, the most options in the file are not evaluated any more.
+
+??? info "Editing the file `/boot/dietpi.text` contents"
+
+    On some hardware options (e.g. Raspberry Pi) the file is located on an own partition which can be accessed on a Windows PC.
+
+    Some hardware options (e.g. NanoPi) do not have this extra partition. In this case the SD card needs to be mounted to be able to access the `dietpi.tfxt` file.
+
+The result of the automated installation can cover the following areas:
+
+- **Network configuration** (e.g. WiFi, network and proxy settings)
+- **System options** (e.g. timezone, hostname, root password)
+- **Software preferences** (e.g. SSH server, file server, web server)
+- **Software options** (e.g. VNC, Nextcloud, DietPi Dashboard)
+- **Automatic software installation** (packages from `dietpi-software`)
+- **User script** pre and post initial installation
+- **Restore** from a previous made system backup
+
+To achieve this automated installation, the following steps need to be conducted in the given order:
+
+1. Flash the boot media (e.g. SD card)
+1. Edit the file `/boot/dietpi.txt` on the boot media
+1. Attach the boot media to your system
+1. Start up the system the first time
+1. Wait for completion of the automated setup
+
+### Options within the file
+
+The options are generally described within the file itself. Basically see the file contents for details about the configuration options.  
+Below, only the most important options are listed. For further options go into the file `dietpi.txt` itself.
+
+#### Network configuration
+
+To achieve the basic network configuration the following options shall be taken into account:
+
+- Run with WiFi: AUTO_SETUP_NET_WIFI_ENABLED, AUTO_SETUP_NET_WIFI_COUNTRY_CODE in combination with `dietpi-wifi.txt`
+- Use a static IP address: AUTO_SETUP_NET_USESTATIC, AUTO_SETUP_NET_STATIC_IP, AUTO_SETUP_NET_STATIC_MASK, AUTO_SETUP_NET_STATIC_GATEWAY, AUTO_SETUP_NET_STATIC_DNS, AUTO_SETUP_DHCP_TO_STATIC
+- Usage of a proxy server: CONFIG_PROXY_ADDRESS, CONFIG_PROXY_PORT, CONFIG_PROXY_USERNAME, CONFIG_PROXY_PASSWORD
+
+#### System options
+
+Many of the system options can be set with the automated installation and can be also changed afterwards via `dietpi-config`:
+
+- Basic settings: AUTO_SETUP_NET_HOSTNAME, AUTO_SETUP_GLOBAL_PASSWORD
+- Localization: AUTO_SETUP_LOCALE, AUTO_SETUP_KEYBOARD_LAYOUT, AUTO_SETUP_TIMEZONE
+- Autostart: AUTO_SETUP_AUTOSTART_TARGET_INDEX, AUTO_SETUP_AUTOSTART_LOGIN_USER
+- Overclocking: CONFIG_CPU_GOVERNOR, CONFIG_CPU_MAX_FREQ, CONFIG_CPU_MIN_FREQ, etc.
+- Auto-updating: CONFIG_CHECK_DIETPI_UPDATES, CONFIG_CHECK_APT_UPDATES
```

Comment:
As mentioned above, wrap settings names into code tags as well, to exclude them from spellchecking and highlight them visually from common text/sentences.

Reply:
Done.